### ISSUE TITLE
Keeper hand blinks on the UI

### ISSUE THREAD

**Issue by akien-mga** [OpenDungeons/OpenDungeons#868]
This is a known issue that comes from the refresh rate of the UI needed to update the new camera minimap. It's fine this way for the current release, but we might want to find a way to improve this for 0.7.0, either by putting the minimap on another UI layer if possible (then the "blinking" will only be when hovering the minimap), or something else.

**Comment by Bertram25** [comment]
That one is indeed not an easy one. IMHO, we should definitely open a thread on ogre about the better way to draw a texture, or a scene on a texture in ogre forums.

I still believe custom low-poly models should be drawn with materials ignoring ambient light in the same scene, with some kind of off-screen rendering technique if possible. But all that is theory until someone puts his/her hands on this.

**Comment by hwoarangmy** [comment]
I agree this one might not be very easy unless some ogre guy pops up and tells us some magic...

**Comment by Bertram25** [comment]
@hwoarangmy @akien-mga  I don't have this issue anymore with the latest build. Can you confirm?

**Comment by hwoarangmy** [comment]
unfortunately, it is still present when the minimap camera is used. It is related somehow to rendering in the texture minimap (maybe because it takes too much time to render).

**Comment by Bertram25** [comment]
> unfortunately, it is still present when the minimap camera is used. It is related somehow to rendering in the texture minimap (maybe because it takes too much time to render).

Arg, you're right. Sorry, I'm still in the process of catching up.
